packages/mail/lib/unsubscribeFromAllNewsletters.js

```diff
@@ -0,0 +1,17 @@
+const MailchimpInterface = require('../MailchimpInterface')
+const { EmailRequiredMailError } = require('./errors')
+const logger = console
+
+module.exports = async ({
+  email
+}) => {
+  if (!email) {
+    throw new EmailRequiredMailError()
+  }
+
+  const mailchimp = new MailchimpInterface({ logger })
+  return mailchimp.updateMember(email, {
+    email_address: email,
+    status: 'unsubscribed'
+  })
+}
```

Comment:
the name of this file is misleading. I would expect it to unsubscribe an email from all interests, instead it unsubscribes an email from mailchimp completely. change to `unsubscribeEmail`